.github/workflows/ci.yml

```diff
@@ -58,13 +58,127 @@ jobs:
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
 
+  unit-tests:
```

Comment:
The job name must be more specific since in the beginning we'll have multiple unit test jobs. Which units tests does this execute exactly? Zeebe's?

Reply:
`search-unit-tests` or `zeebe-unit-tests` would be preferred.

Reply:
No because I want to run all unit-tests (which is essential done here) :)

Reply:
Ah got it, I'm not an expert with which Maven profiles mean what in C8 😀 Then it makes sense - the PR branch `search-ci` lead me to assume you'd add a dedicate new test suite and only want to run that.